topdown/uuid.go

```diff
@@ -31,6 +32,32 @@ func builtinUUIDRFC4122(bctx BuiltinContext, operands []*ast.Term, iter func(*as
 	return iter(result)
 }
 
+func builtinUUIDParse(_ BuiltinContext, operands []*ast.Term, iter func(term *ast.Term) error) error {
+	str, err := builtins.StringOperand(operands[0].Value, 1)
+	if err != nil {
+		return err
+	}
+
+	parsed := uuid.Parse(string(str))
```

Comment:
```
parsed := uuid.Parse(string(str))
if len(parsed) == 0 {
     return nil
}
```
riight, so it would be more explicit to return an error, check for that and then return the undefined? We dont want the parsing to cause an error in the application, so it would just check for an error and not return it. The code above is how it is handled in the topdown function now